code/src/components/MovieDetails.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable max-len */
+/* eslint-disable no-trailing-spaces */
+import React, { useEffect, useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+// import { MOVIE_DETAILS } from 'utils/urls'; - keeping this for knowledge (if someone else solved it as a shortcut)
+
+const MovieDetails = () => {
+  const navigate = useNavigate()
+  const { id } = useParams()
+  const [details, setDetails] = useState({})
+
+  const onBackButtonClick = (e) => {
+    e.preventDefault();
+    navigate(-1)
+  }
+  // const notFoundButtonClick = () => {
+  //   navigate('/404')
+  // }
+
+  // when replacing the MOVIE_DETAILS in the fetch with the actual link, it finally worked
+  useEffect(() => {
```

Comment:
No idea why the MOVIE_DETAILS link wouldn't work, but maybe it has to do with how the Id is passed? We didn't try, so not sure.